criu/util.c

```diff
@@ -795,6 +795,45 @@ struct vma_area *alloc_vma_area(void)
 	return p;
 }
 
+static int __check_cmd_exists(const char *path)
+{
+	return (access(path, F_OK) == 0);
+}
+
+int check_cmd_exists(const char *cmd)
+{
+	int ret = 0;
+	char buf[255];
```

Comment:
It is generally a bad practice to use magic numbers without defining them. We need to use PATH_MAX here.